index.html

```diff
@@ -67,30 +74,55 @@ <h2>
     <h2>
       Model
     </h2>
-    <dl class="definitions" data-sort="" data-cite="vc-data-model">
+
+    <p class="issue" title="Definitions under discussion">
+The goal of the definitions in this section is to re-use or establish
+terminology that is common across a variety of digital credential formats and
+protocols.
+    </p>
+
+    <dl class="definitions" data-sort="" data-cite="vc-data-model-2.0">
       <dt>
         <dfn>Digital credential</dfn>
       </dt>
       <dd>
-        <p>
-          Is a [=verifiable credential=] about a person.
-        </p>
+        A cryptographically secured digital document containing one or more
+        [=claims=] made by an [=issuer=] about one or more [=subjects=]. Examples include
+        ISO mobile driver's licenses [[ISO18013-5]] and W3C [=verifiable
+        credentials=] [[VC-DATA-MODEL-2.0]].
```

Comment:
I'd be hesitant to just say "[=verifiable credentials=]" as it's overly broad, and covers "all the things"... we are just dealing with digital documents that have attributes/claims relating to people (for now).

```suggestion
        [=claims=] made by an [=issuer=] about one or more [=subjects=]. Examples include
        ISO mobile driver's licenses such as those specified in [[?ISO18013-5]].
```

Reply:
> I am also hesitant regarding W3C [=verifiable credentials=]. apple-to-apple comparison here would be mdocs and w3c vcdm and ietf sd-jwt vc

@Sakurann agree. That's exactly how I was feeling about it. 

I think it might be better to drop the example specs until we have the Registry #85 #58... we can add them once we agree on the mandated formats. 

```suggestion
        [=claims=] made by an [=issuer=] about one or more [=subjects=].
```